.gitignore

```diff
@@ -26,6 +26,7 @@ win32ver.rc
 lib*dll.def
 lib*.pc
 *.bc
+src/test/regress/test_times.log
```

Comment:
Maybe put it in https://github.com/citusdata/citus/blob/master/src/test/regress/.gitignore

Reply:
done